content/docs/guide/gamemodes/climb.md

```diff
@@ -11,16 +11,92 @@ tags:
+
+---
+## Advanced Techniques 
+
+### Climb has some techniques of its own rarely seen outside of its gamemode.
+
+**Fast Ladder Climbing**: Ladders can be climbed by "walking into them". If you are directly facing the ladder hold W, if sideways A/D. If you face the ladder diagonally you can climb the ladder both ways. If you hold W and A/D the speeds add onto each other resulting in a much faster climb.    
+
+**Ladder Jumps**: The speed you gain from fast ladder climbing is enough to fling you into the air! This technique does not have anything to do with inputting jump (which just pushes you off the ladder) and is more simillar to surfing. Gain speed with the climb and flick off near the top. Just like surfing, the way you flick off dictactes your trajectory, and whether you trade speed for height and vice versa.  
+
+**Crouch Bhops**: Bhopping starting in a crouch position. It may be required in a map when going underneath a low celling. Doing a normal Bhop and crouching in the air will raise your collision box (you tuck in your legs), so to go underneath a hazard you must start the Bhop crounched and usually stay crouched.  ![image](../../../../assets/Crouch_Bhop.png)  
+Crouching will also limit your ablity to turn in the air and thus gain speed as well. This speed disadvantage is negated if you have already reached the Bhop speedcap on KZT since you will remain at 380 regardless of how much speed you try to gain.  
+
+**Standups**: While in the air, if you crouch close enough to the ground (TODO: get exact units) the engine will not let you uncrouch since you'll clip into the floor, instead it waits until the moment you've hit the ground or more space has opened up beneath you and uncrouches. This behavior gives you a nice grace period to execute a frame-perfect. (or tick-perfect) outcome. (Note that the faster you are falling, the shorter the grace period is).  
+
+**Standup Bhops**: Combines Standups with bhoping, the idea is that the best of both worlds is that you perfectly time and hold your crouch so that you only "tuck your legs" to get the extra distance. Sounds like a tall order but thanks to how "Standups" behave, to get the best outcome you only have to time the beginning of the crouch to be inside this window and the engine does the rest of the timing. This is not a replacement for regular Bhops or Crouch Bhops, since you may prefer to land sooner and jump sooner or need to aviod map hazards. Standup Bhops are only for squeezing extra distance out of a jump in an optimal manner. 
+
+**Double Duck [1.6 Climb only]**: In 1.6 Climb, while on the ground pressing duck and releasing it before you are fully on the ground will cause the game to unduck you prematurely, the result is that you pop 18 units off the ground (very small compared to a jump) you can improve this height by crouching again in the air resulting in being 36 units off the ground (better but still small compared to a jump).  
```

Comment:
> In 1.6 Climb, while on the ground pressing duck and releasing it before you are fully on the ground

you're always on the ground, would be better to say "before you are fully ducked"